### PR TITLE
Change from optional to required for the three attributes under attached_vpc in aws_tgw

### PR DESCRIPTION
**Pull request by zhiqiangjin**
All update blocks in other resources.

### CODE REVIEW

**Comment by arvindsree** [comment]
Update the docs as well to remove empty lists for blocks

File: aviatrix/resource_aviatrix_fqdn.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func resourceAviatrixFQDN() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"gw_name": {
 							Type:        schema.TypeString,
-							Optional:    true,
+							Required:    true,
 							Description: "Name of the gateway to attach to the specific tag.",
 						},
 						"source_ip_list": {
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_fqdn.go]: Should we give default value for this?

Comment (zhiqiangjin) [reply aviatrix/resource_aviatrix_fqdn.go]: No, we should give the name of the gateway. I think it should be required.

Comment (arvindsree) [reply aviatrix/resource_aviatrix_fqdn.go]: i meant for source IP list

Comment (arvindsree) [reply aviatrix/resource_aviatrix_fqdn.go]: default as empty list? just test it if it is required or it takes default as null

Comment (zhiqiangjin) [reply aviatrix/resource_aviatrix_fqdn.go]: It takes default as null. We can leave it as optional.

Comment (arvindsree) [reply aviatrix/resource_aviatrix_fqdn.go]: will code crash for null?

**Pull request merged**